.env.sample

```diff
@@ -24,10 +26,10 @@ REACT_APP_NETWORK=rpc
 REACT_APP_SERVER_URL=http://localhost:8000
 
 # Web3 stuff config
-PRIVATE_KEY=
 NETWORK=rpc
-RPC=http://localhost:8545
-COURT_ADDRESS=0xD833215cBcc3f914bD1C9ece3EE7BF8B14f841bb
-GAS_PRICE=15000000001
 GAS=7000000
+GAS_PRICE=15000000001
 WEB3_POLLING_INTERVAL=10000
+COURT_ADDRESS=0xD833215cBcc3f914bD1C9ece3EE7BF8B14f841bb
```

Comment:
Similar nitpick, but this seems like it could be at the top of the list or etc. since it's very fundamental.